src/SFA.DAS.RoatpGateway.Web/Views/Gateway/Pages/_GatewayClarificationDetails.cshtml

```diff
@@ -15,23 +15,23 @@
             Date of check
         </dt>
         <dd class="govuk-summary-list__value">
-            @Model.OutcomeMadeOn.ToSfaShortDateString()
+            @Model.ClarificationDate.ToSfaShortDateString()
```

Comment:
I'm not sure how well this will play with 2033 (withdrawal) as it uses OutcomeMadeOn